core/web/components/forms/source/mapping.tsx

```diff
@@ -105,6 +113,19 @@ export default function ({ apiVersion, errorHandler, successHandler, query }) {
     if (response?.source) {
       successHandler.set({ message: "Source updated" });
       setSource(response.source);
+
+      // this source can have a schedule, and we have no schedules yet
+      if (scheduleCount === 0 && response.source.scheduleAvailable) {
```

Comment:
If I read it right, the this actually makes a schedule. The [story](https://www.pivotaltracker.com/story/show/172010602) is more about redirecting to that page, not making one explicitly. Or did I read this wrong?

Reply:
yes... but there was another theme to remove as many of the the modals as possible.  Since schedules no longer need tiles, there's no user input needed to create a schedule any more... so there's no schedule/new page.   To be validated later with @andyjih

Reply:
I guess the issue is that there needs to be a schedule to take them to that page for the same reason we need all this draft business?

Reply:
That's what the `createSchedule` method does!